components/RandomMoviePicker/RandomMoviePicker.js

```diff
@@ -0,0 +1,77 @@
+/// This function is rendering through the passed movie data array and is picking one random movie to show.
+
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import { ReadMore } from "../../components/ReadMoreText/ReadMoreText";
+import Header from "../../components/Header/Header";
+import styled from "styled-components";
+
+export default function RandomMoviePicker({ movieArray }) {
+  const [randomPick, setRandomPick] = useState(0); // picks a random number
+  const listItems = movieArray.map((movie) => <li key={movie.id}></li>); // map through the movie Array
+  const productionYear = movieArray[randomPick].release_date.slice(0, 4); // slices down the production year, e.g. 2014-12-04 > 2014
+  useEffect(() => {
+    setRandomPick(Math.floor(Math.random() * listItems.length));
+  }, [listItems.length]); // calculates a random number on a page load
+
+  // return the movie details bassed on the random number
+  return (
+    <>
+      <Header />
+      <SectionStyled aria-label="movie content">
+        <ImageWrap aria-label="Image Wrap Div">
+          <Image
+            src={`https://image.tmdb.org/t/p/w500/${movieArray[randomPick].poster_path}`}
+            alt={movieArray[randomPick].title}
+            width={203}
+            height={304}
+          />
+        </ImageWrap>
+        <h2>{movieArray[randomPick].title}</h2>
+        <MovieTagsUl>
+          <MovieTags>{movieArray[randomPick].runtime} Minuten</MovieTags>
+          <MovieTags>{movieArray[randomPick].genres[0].name}</MovieTags>
+          <MovieTags>Jahr: {productionYear}</MovieTags>
+        </MovieTagsUl>
+        <ReadMore aria-label="mehr lesen oder weniger lesen">
```

Comment:
```suggestion
        <ReadMore aria-label="read more or read less">
```
better to keep everything in english as the screen reader includes a translator :)
Later you could lift up your state and change the aria label depending on IsReadMore being true or false :)

Reply:
done.